modules/lazyload/module.php

```diff
@@ -87,6 +87,20 @@ private function apply_dominant_color_background( $control, $value, $selector )
 		return $control;
 	}
 
+	private function is_document_support_lazyload( $post_id = null ) {
+
+		$document = \Elementor\Plugin::$instance->documents->get( $post_id ?? get_the_ID() );
```

Comment:
Please force supplying the post ID.

```suggestion
	private function is_document_support_lazyload( $post_id ) {

		$document = \Elementor\Plugin::$instance->documents->get( $post_id );
```

Reply:
Done